docs-content/installing.html.md.erb

```diff
@@ -1,28 +1,23 @@
 ---
-title: Installing and Configuring YOUR-PRODUCT-NAME
+title: Installing and Configuring Datadog Application Monitoring for PCF
 owner: Partners
 ---
 
-This topic describes how to install and configure YOUR-PRODUCT-NAME.
+This topic describes how to install and configure Datadog Application Monitoring for PCF.
 
-##<a id='install'></a> Install and Configure YOUR-PRODUCT-NAME
+##<a id='install'></a> Install and Configure Datadog Application Monitoring for PCF.
 
-1. Download the product file from Pivotal Network.
+1. Download the product file for the **Datadog Application Monitoring for PCF** from Pivotal Network.
```

Comment:
Would the BOSH Health Monitor also be required as the first step here? (similar to our first tile: https://docs.pivotal.io/partners/datadog/installing.html)